### PR TITLE
Fixed #26962 -- Doc'd running migrations in transactions.

### PR DESCRIPTION
**Pull request by caioariede**
Discussion: https://code.djangoproject.com/ticket/26962

The text was copied (with small modifications) from a [how to section](https://docs.djangoproject.com/en/dev/howto/writing-migrations/#non-atomic-migrations)

---

Screen Shot 2020-10-28 at 09 21 29

### CODE REVIEW

File: docs/topics/migrations.txt [hunk 1]
```diff
@@ -305,6 +305,27 @@ migrations are treated no differently from any other migration.
...
+    from django.db import migrations
+
+    class Migration(migrations.Migration):
+        atomic = False
+
+Within such a migration, all operations are run without a transaction. It's
+possible to execute parts of the migration inside a transaction using
+:func:`~django.db.transaction.atomic()` or by passing ``atomic=True`` to
+:class:`~django.db.migrations.operations.RunPython`.
+
```
Comment (felixxm) [inline docs/topics/migrations.txt:328]: ` .. _migration-history-consistency: ` should be above the related section.

File: docs/topics/migrations.txt [hunk 2]
```diff
@@ -305,6 +305,27 @@ migrations are treated no differently from any other migration.
 
 .. _migration-history-consistency:
 
+Non-atomic migrations
+---------------------
```
Comment (felixxm) [inline docs/topics/migrations.txt:309]: I think this should be a main topic, maybe below the `Workflow`, e.g.
```suggestion
Transactions
============
```

File: docs/topics/migrations.txt [hunk 3]
```diff
@@ -305,6 +305,27 @@ migrations are treated no differently from any other migration.
...
+
+.. attribute:: Migration.atomic
```
Comment (felixxm) [inline docs/topics/migrations.txt:311]: I would expect a general topic about handling transactions by migrations, not a documentation of `Migration.atomic`, maybe:
```
On databases that support DDL transactions (SQLite and PostgreSQL), all
migration operations will run inside a single transaction by default. In
contrast, if a database don't support DDL transactions (e.g. MySQL, Oracle)
then all operations will run without a transaction.

You can prevent a migration from running in a transaction by setting the
``atomic`` attribute to ``False``. See :ref:`non-atomic-migrations` for
details.
```

**Review by felixxm — COMMENTED** [review]
@caioariede Thanks :+1:

File: docs/topics/migrations.txt (hunk 2 above)
Comment (caioariede) [reply docs/topics/migrations.txt]: Agreed!

**Review by felixxm — APPROVED** [review]
@caioariede Thanks for updates :+1:

**Review by carltongibson — APPROVED** [review]
Perfect. Nice. Thank you!

**Pull request merged**